backend/src/channels/channels.controller.spec.ts

```diff
@@ -19,9 +28,72 @@ describe('ChannelsController', () => {
     }).compile();
 
     controller = module.get<ChannelsController>(ChannelsController);
+    service = module.get<ChannelsService>(ChannelsService);
   });
 
   it('should be defined', () => {
     expect(controller).toBeDefined();
   });
+
+  describe('findAll()', () => {
+    it('should return an array of channels', async () => {
+      const expected = new PageDto(
+        [],
+        new PageMetaDto(new PageOptionsDto(), 10),
```

Comment:
```suggestion
        new PageMetaDto(new PageOptionsDto(), takeDefault),
```
`takeDefault`, already a `const` in `src/common/dto/page-options.dto.ts` would be exported (maybe with a better name).

Reply:
Good idea, I exported takeDefault but left the name the same for now. Let me know what you think